install/Knative-with-ICP.md

```diff
@@ -0,0 +1,301 @@
+# Knative Install on IBM Cloud Private
+
+This guide walks you through the installation of the latest version of
+[Knative Serving](https://github.com/knative/serving) using pre-built images and
+demonstrates creating and deploying an image of a sample `hello world` app onto
+the newly created Knative cluster.
+
+You can find [guides for other platforms here](README.md).
+
+## Before you begin
+
+Knative requires an [IBM Cloud Private](https://www.ibm.com/cloud/private) cluster v3.1.1. See [Installing IBM Cloud Private Cloud Native, Enterprise, and Community editions]((https://www.ibm.com/support/knowledgecenter/SSBS6K_3.1.1/installing/install_containers.html)) in the IBM Knowledge Center for install instructions.
+
+### Step 1: Install Docker for your boot node only
```

Comment:
I think we do not need step 1 to step 5, just navigate this to IBM Cloud Private KC is good enough.

Also please navigate to IBM Cloud Private CE version, as EE version is not free.

Reply:
If pointing out to the IBM Cloud Private KC includes only the steps covered here in 1-5, then yes, I agree that simply linking to those steps is ok.

BUT, if the linked page contains a bunch of unnecessary steps or requirements, then its a poor user experience to link out to a separate set of instructions and therefore, its better to include steps 1-5 here.